src/pytorch_lightning/plugins/precision/deepspeed.py

```diff
@@ -97,24 +93,19 @@ def backward(
                 "You have overridden the `LightningModule.backward` hook but it will be ignored since DeepSpeed handles"
                 " the backward logic internally."
             )
+        _, _, *args = args
```

Comment:
Not sure fan of those args unpacking. This is easy to forget and break stuff.

Reply:
what should I do instead?

Reply:
I prefer not to leak them to Lite as they aren't required there.